.husky/pre-commit

```diff
@@ -0,0 +1 @@
+npx lint-staged
```

Comment:
Have you set this to be executable? I know that's a common issue from other repos.

Reply:
Of course, Husky hooks are prepared to avoid permitting issues.

![image](https://github.com/preactjs/preact/assets/116913262/0ec13f73-db40-4f82-bc1f-d336158ad3a4)

Reply:
I'd rather you just made the file executable. `prepare` is going to run extremely often and we want to minimize what's in it.